osu.Game.Rulesets.Osu/Objects/Drawables/DrawableHitCircle.cs

```diff
@@ -42,7 +42,7 @@ public DrawableHitCircle(OsuHitObject h) : base(h)
 
                         Judgement.PositionOffset = Vector2.Zero; //todo: set to correct value
                         UpdateJudgement(true);
-                        return true;
+                        return !(Parent is DrawableSlider);

```

Comment:
What about if we also change it so that `DrawableSlider` consumes actions if the cursor is over it?